.env.example

```diff
@@ -0,0 +1,9 @@
+
+DATABASE_CONNECTION_URL=
+
+JWT_KEY=
```

Comment:
I don't think JWT is needed as of now for this project. Only add those environment variables which are currently required, otherwise, it will create confusion for other developers.

Reply:
Removed